cinema/views.py

```diff
@@ -1 +1,65 @@
-# write views here
+from rest_framework import viewsets
+from cinema.models import (CinemaHall,
+                           Genre,
+                           Actor,
+                           Movie,
+                           MovieSession)
+
+from cinema.serializers import (GenreSerializer,
+                                ActorSerializer,
+                                CinemaHallSerializer,
+                                MovieSerializer,
+                                MovieListSerializer,
+                                MovieSessionSerializer,
+                                MovieSessionListSerializer,
+                                MovieRetrieveSerializer,
+                                MovieSessionRetrieveSerializer)
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code.

Example:
```python
from taxi.models import (
Driver,
Manufacturer,
Car
)
```
Read more about topic [here](https://peps.python.org/pep-0008/#imports)